react-app/src/components/Nav/Nav.tsx

```diff
@@ -15,7 +15,7 @@ export default function Nav() {
 
     return (
         // @ts-ignore
-        <Header aria-label="IBM Platform Name">
+        <Header aria-label="Open Harvest">
```

Comment:
Apparently we're cool enough to have it be one complete word now. Which I think what the brand is. I think one of Daniel's later commits address it. So you'll have to rebase to get those changes

Reply:
Oh yeah, I saw it, I will change it. Yes! We are cool 😎